egs2/aishell/asr1/README.md

```diff
@@ -0,0 +1,37 @@
+## Environments
+- date: `Sun Feb  2 02:03:55 CST 2020`
+- python version: `3.7.3 (default, Mar 27 2019, 22:11:17)  [GCC 7.3.0]`
+- espnet version: `espnet 0.6.0`
+- pytorch version: `pytorch 1.1.0`
+- Git hash: `e0fd073a70bcded6a0e6a3587630410a994ccdb8` (+ fixing https://github.com/espnet/espnet/pull/1533)
+  - Commit date: `Sat Jan 11 06:09:24 2020 +0900`
+
+## asr_train_asr_rnn_new_fbank_pitch_char
+### WER
+
```

Comment:
You can just remove the WER result as it corresponds to the sentence-level error for Mandarin

Reply:
You are right. I will remove it.

Reply:
Hello, may I use the asr model trained based on the fbank_pitch feature in aishell under the egs2 directory of espnet. If you want to use asr_interface.py under espnet2 for speech recognition, what kind of changes need to be made to the provided python code?

Reply:
Hi @BlueSkyForever, I don't think current espnet2 supports extracting pitch from raw waveform. As you may know, in espnet1, the pitch feature is extracted using the Kaldi command.

Reply:
Okay, thanks for your answers.